src/Pos/Block/Pure.hs

```diff
@@ -584,4 +625,4 @@ verifyBlocks curSlotId verifyNoUnknown bvd initLeaders = view _3 . foldl' step s
                 , vbpMaxSize = Just (bvdMaxBlockSize bvd)
                 , vbpVerifyNoUnknown = verifyNoUnknown
                 }
-        in (newLeaders, Just $ getBlockHeader blk, res <> verifyBlock vbp blk)
+        in (newLeaders, newPsks, Just $ getBlockHeader blk, res <> verifyBlock vbp blk)
```

Comment:
I still don't understand anything.

> the initial set of certificates valid relatively to the beginning of blocks list

What does it mean? Shouldn't it be from GState?